README.rst

```diff
@@ -23,4 +23,27 @@ The primary use cases are:
 
 
 `The documentation <https://jupyter-server-proxy.readthedocs.io/>`_
-contains information on installation & usage.
\ No newline at end of file
+contains information on installation & usage.
+
+====================
+Install
+====================
+
+pip
+---
+
+pip install jupyter-server-proxy
```

Comment:
If you indent this line (and the others like it) they will show up as `code` (I think), just to drive home the point to the reader that this is something to put in a terminal.